content/docs/guides/branching-github-actions.md

```diff
@@ -113,6 +114,58 @@ jobs:
 
 This Action has no outputs.
 
+## Reset from parent action
+
+This GitHub Action resets a child branch with the latest data from its parent branch.
+
+> **Info**
+> The source code for this action is available on [GitHub](https://github.com/neondatabase/reset-branch-action).
+
+### Prerequisites
+
+- Using this action requires a Neon API key. For information about obtaining an API key, see [Create an API key](/docs/manage/api-keys#create-an-api-key).
+- Add your Neon API key to your GitHub Secrets:
+    1. In your GitHub repository, go to **Settings** and locate **Secrets** at the bottom of the left sidebar.
+    2. Click **Actions** > **New Repository Secret**.
+    3. Name the secret `NEON_API_KEY` and paste your API key in the **Secret** field.
+    4. Click **Add Secret**.
+
+### Example
+
+The following example demonstrates how to reset a branch in your Neon project:
+
+```yaml
+name: Reset Neon Branch with GitHub Actions Demo
+run-name: Reset a Neon Branch 🚀
+jobs:
+  Reset-Neon-Branch:
+    uses: neondatabase/reset-branch-action@beta
```

Comment:
we probably should release v1 before actually publishing this, I will do it in the beginning of next week, so far I didn't receive any actionable feedback